src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -1,39 +1,81 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { addComment, getPostComments } from '../../api/comments';
+import { Comment } from '../../react-app-env';
 import './NewCommentForm.scss';
 
-export const NewCommentForm: React.FC = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <textarea
-        name="body"
-        placeholder="Type comment here"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <button
-      type="submit"
-      className="NewCommentForm__submit-button button"
+type Props = {
+  postId: number,
+  commentId: number
+  setComments: (response: Comment[]) => void
+};
+
+export const NewCommentForm: React.FC<Props> = ({ postId, commentId, setComments }) => {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [text, setText] = useState('');
+
+  const handlerSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    addComment({
+      postId,
+      name,
+      email,
+      body: text,
+      id: commentId + 1,
```

Comment:
id is set to comment automatically, no need to pass it, all you need to pass here
<img width="876" alt="image" src="https://user-images.githubusercontent.com/64437769/160786867-996faa68-baee-43fd-89a8-dc8dc1fca082.png">